crates/driver/src/tests/cases/negative_scores.rs

```diff
@@ -23,12 +23,32 @@ async fn no_valid_solutions() {
 
 #[tokio::test]
 #[ignore]
-async fn one_valid_solution() {
+async fn one_valid_solution_with_solver_score() {
```

Comment:
@nlordell are these the driver specific e2e tests we talked about yesterday?

Reply:
Yesterday I mentioned that I will first add these driver tests and then see if it's enough or not. IMO, there is no need to add [full e2e](https://github.com/cowprotocol/services/tree/main/crates/e2e) test.